A3-Antistasi/REINF/Building/build.sqf

```diff
@@ -141,6 +141,18 @@ switch build_type do
 		_classX = "Land_PillboxBunker_01_big_F";
 		build_cost = 300;
 		};
+		case "SBS":
```

Comment:
Got a few different indentation issues in this section. Both case statements as well as the build cost at the bottom